test/client/api.js

```diff
@@ -427,4 +426,24 @@ ClientApi.heartbeat = function (origin) {
   return (new ClientApi(origin)).doRequest('GET', origin + '/__heartbeat__')
 }
 
+function getQueryString (options) {
+  var qs = ''
```

Comment:
Would it be easier to set a default of "?" and then not have to worry about managing "?" vs "&" below?

```js
var qs = '?'

if (options.keys) {
  qs += 'keys=true&'
}

if (options.serviceQuery) {
  qs += 'service=' + options.serviceQuery
}

return qs
```

Or possibly easier with https://nodejs.org/api/querystring.html#querystring_querystring_stringify_obj_sep_eq_options

```js
function getQueryString (options) {
  return '?' + querystring.stringify(options)
}
```